docs/guides/databases/redis/install-and-configure-redis-on-ubuntu-20/index.md

```diff
@@ -0,0 +1,269 @@
+
+## Configuring an Access Control List (ACL) for Redis
+
+In a multi-user environment, we highly recommend you configure an ACL. An ACL restricts privileges and potentially dangerous commands to a subset of the users. You can create additional users in the `redis.conf` file. Each user directive must contain a username, the keyword `on` or `off` to enable or disable the entry, a set of command permissions and restrictions, a set of key permissions, and the password (with a preceding `>` symbol). To indicate all commands, use `@all`. Use `allkeys` to reference all key-value pairs. Redis parses each user directive from left to right, so the order of permissions and restrictions is important.
+
+This is only a brief introduction to this topic. Create users with memorable names, strong passwords, and appropriate permissions when building your own ACL. A complete explanation of all the ACL options is available on the [*Redis ACL page*](https://redis.io/topics/acl).
+
+1.  Display the current ACL configuration with the `ACL LIST` command. It currently shows only the default user and their privileges.
+
+        redis-cli
+        ACL LIST
+2.  Edit the `redis.conf` file and add user directives for two users. The first directive adds `user2` and assigns the password `user2pass`, along with access to all commands and keys. The second user directive, for `user3`, is similar except this user cannot run `SET` commands.
+    {{< file "/etc/redis/redis.conf" >}}
+user user2 +@all allkeys on >user2pass
+user user3 +@all -SET allkeys on >user3pass
+{{< /file >}}
+3.  Restart Redis to force the changes to take effect.
+
+        sudo systemctl restart redis.service
+4.  Re-enter the Redis CLI and log in as `user2` (authenticate with the `AUTH` command). Execute a `SET` command. Redis now returns an `OK` response.
+
+        redis-cli
+        AUTH user2 user2pass
+        SET server:name "fido2"
+5.  Exit and then re-enter the Redis CLI, authenticating as `user3`. Attempt to execute another `SET` to change the same key to a new value.
+
+        redis-cli
+        AUTH user3 user3pass
+        SET server:name "fido3"
+    This attempt fails and returns a permission error.
+
+        (error) NOPERM this user has no permissions to run the 'set' command or its subcommand
+6.  Attempt to retrieve the value of this same key with a `GET` command. Redis now returns the value.
+
+        GET server:name
+
+{{< note >}}
+You can also create users through the `redis-cli` interface using the `ACL SET USER` command, or through a separate ACL file. See the [*Redis ACL page*](https://redis.io/topics/acl) for more information.
+
+In earlier versions of Redis, administrators could rename and therefore hide powerful commands using `rename-command` directives in `redis.conf`. With the introduction of the ACL feature, this directive is no longer recommended. However, it is still available for backwards compatibility.
+{{< /note >}}
+
+## Configuring Redis Persistence
+
```

Comment:
Maybe a brief sentence or two on  why persistence is so important for Redis (because everything is stored in memory, etc)